valohai_yaml/objs/pipelines/deployment_node.py

```diff
@@ -20,8 +20,9 @@ def __init__(
         actions: Optional[List[NodeAction]] = None,
         endpoints: Optional[List[str]] = None,
         aliases: Optional[List[str]] = None,
+        continue_on_error: Optional[bool] = False,
```

Comment:
The value can't be true/false/none, can it?

```suggestion
        continue_on_error: bool = False,
```